client/src/utils/is-new-responsive-web-design-cert.ts

```diff
@@ -0,0 +1,5 @@
+import { SuperBlocks } from '../../../config/certification-settings';
```

Comment:
My 2 cents: I would prefer these tiny functions going in the `utils/index.ts` file, as opposed to their own.

Reply:
I'm the opposite tbh - I like having them in a separate file. Easier to find.